### PR TITLE
Show Application Counts by default

### LINKED ISSUE #1820: Remove showCounts parameter after implements showCounts API
https://github.com/pipe-cd/pipe/blob/5fcca1a3330cede756b4b53f82a9daa2e01e77a5/pkg/app/web/src/pages/applications/index.tsx#L54-L57

### PR DESCRIPTION
**Pull request by cakecatz**
**What this PR does / why we need it**:

If you get application counts with no data, you will get an Error Toast but it is correct that it does not exist in the initial state, so I did not give an error.

**Which issue(s) this PR fixes**:

Fixes #1820

**Does this PR introduce a user-facing change?**:

```release-note
Show Application Counts by default
```

### CODE REVIEW

File: pkg/app/web/src/mocks/services/insight.ts [hunk 1]
```diff
@@ -28,6 +29,11 @@ export const getInsightApplicationCountHandler = createHandler<
   return response;
 });
 
+export const getInsightApplicationCountNotFound = createHandlerWithError(
```
Comment (nghialv) [inline pkg/app/web/src/mocks/services/insight.ts:32]: Where is this used?

Comment (cakecatz) [reply pkg/app/web/src/mocks/services/insight.ts:32]: @nghialv 
Oh, I added it even though I haven't used it yet.
I'll remove it.

Comment (nghialv) [reply pkg/app/web/src/mocks/services/insight.ts:32]: 👍

File: pkg/app/web/src/mocks/services/insight.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { StatusCode } from "grpc-web";
```
Comment (nghialv) [inline pkg/app/web/src/mocks/services/insight.ts:1]: This is imported but not used, right?

Comment (cakecatz) [reply pkg/app/web/src/mocks/services/insight.ts]: Oh yeah, I removed that. Thank you.

**Pull request merged**